sections/featured-collection.liquid

```diff
@@ -55,7 +55,7 @@
         or section.settings.show_description
         and section.settings.collection.description != empty
       -%}
-        <div class="collection__description {{ section.settings.description_style }}">
+        <div class="collection__description {{ section.settings.description_style }} rte">
```

Comment:
Kinda similar to my first comment, here we will need to call the `component-rte.css` file to be sure the styling is applied. 
But maybe we move RTE styles to base.css instead. I'm not sure what's better. Calling multiple assets or having a huge file tbh. 
Maybe for now we keep it as is and call that css file from the top of this file. But that could be an asset we load in theme.liquid as rte settings or rte from the admin is most likely on every page of the theme 🤔

Reply:
I'm leaning towards calling multiple assets but keeping the base file clean.